src/commands/deploy/deploy.ts

```diff
@@ -379,6 +383,12 @@ async function deployToSasjs(target: Target, sasjs?: SASjs) {
       )
     }
   }
+
+  if (streamConfig?.streamWeb) {
+    const appLoc = target.appLoc.replace(/\ /g, '%20')
```

Comment:
I'm not sure what special characters we'd choose - they could also cause problems in SAS/Drive, Metadata.  I think it would be better to restrict the characters that can be used for an an AppLoc in the JSON schema

Reply:
Space is definitely an exception, as some default SAS folders have spaces (such as "My Folder")

Reply:
How about using `encodeURI`
`encodeURI(target.appLoc)`

Reply:
that should do the job

Reply:
fixed.